packages/core/components/Puck/components/Preview/index.tsx

```diff
@@ -2,15 +2,25 @@ import { DropZone, dropZoneContext } from "../../../DropZone";
 import { rootDroppableId } from "../../../../lib/root-droppable-id";
 import { useCallback, useContext } from "react";
 import { useAppContext } from "../../context";
+import {
+  DefaultComponentProps,
+  DefaultRootProps,
+} from "../../../../types/Config";
 
 export const Preview = ({ id = "puck-preview" }: { id?: string }) => {
   const { config, dispatch, state } = useAppContext();
 
   const Page = useCallback(
-    (pageProps) =>
-      config.root?.render
-        ? config.root?.render({ ...pageProps, editMode: true })
-        : pageProps.children,
+    (pageProps: DefaultRootProps) => {
+      return config.root?.render
+        ? config.root?.render({
+            ...pageProps,
+            id: "puck-root",
+            editMode: true,
+            puck: { renderDropZone: DropZone },
+          })
+        : pageProps.children;
+    },
```

Comment:
So to check, this ensures that components built using the renderDropZone functionality can still render in Preview mode following `v0.13.0`?

Reply:
Correct